Dockerfile

```diff
@@ -12,5 +12,12 @@ RUN apt-get -q update                   \
     && apt-get -q clean                 \
     && rm -rf /var/lib/apt/lists/*
 
-ENTRYPOINT ["/bin/bash"]
+RUN curl -sL -o dumb-init.deb                                                                   \
+        https://github.com/Yelp/dumb-init/releases/download/v1.0.0/dumb-init_1.0.0_amd64.deb    \
+    && dpkg -i dumb-init.deb                                                                    \
+    && rm -f dumb-init.deb
+
+# "--single-child" is required because of
+# https://github.com/Yelp/dumb-init/issues/51
+ENTRYPOINT ["dumb-init", "--single-child"]
```

Comment:
https://github.com/lynxtp/docker-ubuntu/issues/12 has been created to track this problem.